controls/mysql.sp

```diff
@@ -0,0 +1,89 @@
+locals {
+  mysql_common_tags = merge(local.thrifty_common_tags, {
+    service = "mysql"
+  })
+}
+
+benchmark "mysql" {
+  title         = "MySQL Checks"
+  description   = "Thrifty developers checks old MySQL DB systems which were created over 90 days ago."
+  documentation = file("./controls/docs/mysql.md")
+  tags          = local.mysql_common_tags
+  children = [
+    control.mysql_db_system_age_90,
+    control.mysql_db_system_low_connection_count
+  ]
+}
+
+control "mysql_db_system_age_90" {
+  title       = "MySQL DB systems created over 90 days ago should be reviewed"
+  description = "MySQL DB systems created over 90 days ago should be reviewed and deleted if not required."
```

Comment:
If there aren't any equivalent settings for reserved instances, or another cost saving measure for long running instances, I think we should remove this control. Having a DB system running for 90 days ago or more alone isn't enough to recommend a deletion.